docs/admin/serverless/kourier_networking/README.md

```diff
@@ -1,5 +1,5 @@
 # Deploy InferenceService with Alternative Networking Layer
-KServe v0.9 and prior versions create the top level `Istio Virtual Service` for routing to `InferenceService` components based on the virtual host or path based routing.
+KServe creates the top level `Istio Virtual Service` for routing to `InferenceService` components based on the virtual host or path based routing.
```

Comment:
it seems like saying 0.9 and prior is kind of not useful right? means every version. and it doesn't change with 0.10. so that we dont have to update this with each version update, we can just say `kserve creates` until that changes.

Reply:
unless im reading this wrong. the next sentence is that "Now kserve provides an option..." but kserve is at v0.9 so im confused a little by what this means.

Reply:
should it maybe say "Starting with v0.9, Kserve provides an option...."

Reply:
im not 100% sure this started with v0.9, @yuzisun can you confirm?